sdk/template/template/test/public/conditionals.spec.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/* eslint-disable no-invalid-this */
+
+import { isNode } from "@azure/core-http";
+import { assert } from "chai";
+import { isLiveMode } from "@azure/test-utils-recorder";
+
+describe("Tests with conditionals", function() {
+  it("should test A #node", function(): void {
```

Comment:
Do you really need these hashtags?
Unless you're suggesting to enforce the user to add this `#node` in their `package.json` as part of their test script, this doesn't have a meaning. 
I'd suggest you remove the hashtag.

Reply:
Sounds good! I'll remove them.